source/news/2016-03-16.html.markdown

```diff
@@ -0,0 +1,98 @@
+---
+title: Announcing dry-rb
+date: 2016-03-16
+---
+
+We are thrilled to announce the official launch of [dry-rb](https://github.com/dry-rb) and this brand new website! The dry-rb organization was established on GitHub last year by Andy Holland. He created a couple of small and very useful gems, namely `dry-container` and `dry-configurable`. The organization has since grown very quickly, with more people joining the effort, and today we already have 9 gems released on RubyGems.org and more in the works.
+
+dry-rb is here to help usher in a new phase of Ruby development. We value good encapsulation, reusability and non-intrusiveness. Our gems are small and focused on solving specific problems. They work with each other very well, and can also be used in existing projects.
+
+## New gem releases
+
+Apart from the release of this website, we'd like to announce big releases of [dry-validation](/gems/dry-validation) and [dry-types](/gems/dry-types). Both gems went through major refactorings and come with many great new features, bug fixes and internal improvements.
+
+### dry-validation 0.7.0
+
+This release comes with a redesigned predicate logic engine, provided by dry-logic, and has lots of new features which make defining validation schemas very simple. The DSL is more concise and expressive: you can use convenient macros to reduce the boilerplate, use nested schemas which allow re-using existing schemas, and define high-level rules which depend on one or more values. We also have new interfaces for configuring external dependencies that a schema might need to apply its predicates.
```

Comment:
Would change the last line to `We also have new interfaces for configuring external dependencies that a schema might need in order to apply its predicates.`

I just read it wrong the first time and think that this flows better.